buddychatcli/ParticipantHelper.cs

```diff
@@ -127,45 +90,74 @@ public List<Participant> createParticipantDataFromNewSessionCSVFileAndMergeWithE
-                     {
-                        string intro = fields[8];
-                         participantDataDict.Add(KEY_INTRO, intro);
-                    }
+            Dictionary<string, string> participantDataDict = new Dictionary<string, string>();
 
-                        if (!string.IsNullOrEmpty(fields[9]))
-                        {
-                            string question1 = fields[9];
-                            string answer1 = fields[10];
-                            participantDataDict.Add(question1, answer1);
-                        }
+            if (!string.IsNullOrEmpty(fields[pronounsFieldNum])) //4
+            {
+                string pronouns = fields[pronounsFieldNum]; //4
+                participantDataDict.Add(KEY_PRONOUNS, pronouns);
+            }
 
-                        if (!string.IsNullOrEmpty(fields[11]))
-                        {
-                            string question2 = fields[11];
-                            string answer2 = fields[12];
-                            participantDataDict.Add(question2, answer2);
-                        }
+            if (!string.IsNullOrEmpty(fields[introFieldNum])) //6
+            {
+                string intro = fields[introFieldNum];
+                participantDataDict.Add(KEY_INTRO, intro);
+            }
 
-                        if (!string.IsNullOrEmpty(fields[13]))
-                        {
-                            string question3 = fields[13];
-                            string answer3 = fields[14];
-                            participantDataDict.Add(question3, answer3);
-                        }
-                        person.data = participantDataDict;
-                        existingParticipants.Add(person);
-                }
+            if (!string.IsNullOrEmpty(fields[question1FieldNum])) //7
+            {
+                string question1 = fields[question1FieldNum]; //7
+                string answer1 = fields[question1FieldNum+1]; //8
+                participantDataDict.Add(KEY_QUESTION1, question1);
+                participantDataDict.Add(KEY_ANSWER1, answer1);
             }
-            return existingParticipants;
+
+            if (!string.IsNullOrEmpty(fields[question2FieldNum]))
+            {
+                string question2 = fields[question2FieldNum];
+                string answer2 = fields[question2FieldNum+1];
+                participantDataDict.Add(KEY_QUESTION2, question2);
+                participantDataDict.Add(KEY_ANSWER2, answer2);
+            }
+
+            if (!string.IsNullOrEmpty(fields[question3FieldNum]))
+            {
+                string question3 = fields[question3FieldNum];
+                string answer3 = fields[question3FieldNum+1];
+                participantDataDict.Add(KEY_QUESTION3, question3);
+                participantDataDict.Add(KEY_ANSWER3, answer3);
+            }
+
+            // Creating Person object
+            Participant participant = new Participant();

```

Comment:
FYI you can do this syntax: 
Participant participant = new Participant {
    email = email,
    name = name,
    ...
}